CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   [#607](https://github.com/greenbone/openvas/pull/607)
 - Rename some nasl functions and func parameters for consistency and fix byte order issue in get_ipv6_element. [#613](https://github.com/greenbone/openvas/pull/613)
 - Change log level from debug to message to show max_host and max_scan during scan start. [#626](https://github.com/greenbone/openvas/pull/626)
+- Changed the redis-openvas.conf, so that it is compliant with the 5.0+ version(s) of redis. [#641](https://github.com/greenbone/openvas/pull/641)
```

Comment:
Is there any info about backwards compatibility of these changes available?

If not this might require an update of the INSTALL.md because currently Redis 3.2+ is listed there as a requirement.

Reply:
I am personally in favor of updating the `INSTALL.md`, since we have for the build requirements Debian **Buster**, and in the [debian package archive](https://packages.debian.org/buster/database), `redis 5.0+` is listed as the available version.